src/scripts/links.js

```diff
@@ -1,10 +1,9 @@
 import { navigate } from './router';
 
-const planetsLink = document.getElementById('planets-link');
-planetsLink.onclick = () => navigate('/planets');
-
-const fallingLink = document.getElementById('falling-link');
-fallingLink.onclick = () => navigate('/');
-
-const pendulumsLink = document.getElementById('pendulums-link');
-pendulumsLink.onclick = () => navigate('/');
+window.addEventListener('click', e => {
```

Comment:
I really like this idea - simpler to set up than we used to have. However, it adds a lot of overhead (every time the user touches the screen, this is called)

Seems like we were planning to move these links to a separate page in the end, in which case this wouldn't benefit us.

Reply:
That seems like a great solution to me! I didn't even know `document.links` existed